src/components/checkbox/checkbox.scss

```diff
@@ -14,10 +14,21 @@
     flex-wrap: wrap;
 
     .mdc-checkbox {
+        .mdc-checkbox__native-control {
+            &[disabled],
+            &:enabled {
+                &:not(:checked):not(:indeterminate) {
+                    ~ .mdc-checkbox__background {
+                        background-color: rgb(var(--contrast-300));
+                    }
+                }
+            }
+        }
+
         &.mdc-checkbox--invalid {
             .mdc-checkbox__native-control:enabled:not(:checked):not(:indeterminate)
                 ~ .mdc-checkbox__background {
-                border-color: var(--lime-error-text-color);
+                border-color: var(--lime-red-text-color);
```

Comment:
Hmm… isn't "error" the one we should use for invalid stuff? Or rather, maybe we should create a separate variable that's called something with "invalid". Generally, we want to avoid using the names of colours in variables like this, since, if we one day decide that we want to use a different colour for invalid, say purple for example, it would make no sense for it to be called `--lime-red-text-color`.

Reply:
this must have happened by accident  :)

Reply:
yes, you are right, I have changed it accidentally by testing something... fixed now